test/settings.gradle.kts

```diff
@@ -9,24 +9,26 @@
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied.
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
 
 includeBuild("../packages")
 
+include("local")
```

Comment:
What was your thinking here? To have a `local` and `sync` package with completely separate tests? Or let a `sync` module depend on the `local` module?

In Java we run the local tests for both sync and local builds as they are using two different `.so` files. It does feel a bit extreme, but I suspect we need to do something similar here to be sure we actually test what we ship?

Reply:
I advices that we should have a separate module for `base` and `sync`. Didn't know that we actually did `base` test on `sync` too. If we initially ship the same `.so`-file it isn't blocking, but it would be a good idea to evaluate if we need to rethink the split.